graphics/DiffPDF/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           qmake 1.0
+
+name                DiffPDF
+version             2.1.3
+categories          graphics
+platforms           darwin
+license             GPL-2+
+maintainers         {gmail.com:davide.liessi @dliessi} openmaintainer
+description         Compare two PDF files textually or visually
+long_description    ${name} can compare two PDF files. \
+                    It offers three comparison modes: \
+                    Words, Characters, and Appearance.
+homepage            http://www.qtrac.eu/diffpdf-foss.html
+master_sites        http://www.qtrac.eu/
+
+distname            [string tolower ${name}-${version}]
+
+checksums           sha1    663ecb7666a4b7d6fbd1a37327fd1f895fae69b8 \
+                    rmd160  12a5f6921b8a53d98d9786af25e408e27611f56f \
+                    sha256  c6142ee038a78108397f45b0c456dca7a4fe1d75250f21a514a134101d322433 \
+                    size    91856
+
+depends_lib-append  port:poppler-qt4-mac
+configure.env-append PKG_CONFIG_PATH=${prefix}/libexec/poppler-qt4-mac/lib/pkgconfig
```

Comment:
Use `configure.pkg_config_path-append` instead of setting the environment variable manually. There is no equivalent for the build phase so if this project really does run pkg-config at build time rather than just at configure time then you do still need to set the variable in the build phase, however you could use the `${configure.pkg_config_path}` variable to avoid repeating yourself.

Reply:
Done, and indeed it's only needed in the configure phase.